marimbabot_behavior/src/marimbabot_behavior/interpreter.py

```diff
@@ -55,12 +55,12 @@ def read_notes(notes) -> HitSequenceGoal:
 
     for note in midi_stream.instruments[0].notes:
         name = pm.note_number_to_name(note.pitch)
+        normalizer = 40
```

Comment:
Tested with normalization value.
Minimum loudness was 0.15 and maximum was 0.67, on the other hand we observe maximum loudness as 1.0 when we pass the "mercato" string.
30 sounded little more louder since we get less loud sound when we increase the normalize value. But 40 also works.

Reply:
Good to merge. Tests were successfull